packages/telegraf/extra/telegraf.ps1

```diff
@@ -0,0 +1,81 @@
+ param (
+    [string]$pkg_inst_dpath,
+    [string]$local_ip
+ )
+
+$conf = "$pkg_inst_dpath\etc\telegraf.conf"
+Add-Content $conf '
```

Comment:
My expectation is to use this config file, that has been already merged:
https://github.com/dcos/dcos/blob/master/gen/dcos-config-windows.yaml#L90
Why is now a duplicated config being proposed to be added?

Reply:
fixed ad b9faf13fdebbc73a355ca570cb4295ce0a37d372. We use the telegraf.conf from https://github.com/dcos/dcos/blob/master/gen/dcos-config-windows.yaml#L90